de-de/geaenderte-optionen.md

```diff
@@ -0,0 +1,89 @@
+| [MC-199467](https://bugs.mojang.com/browse/MC-199467) | Manche Animationen hören nach gewisser Zeit auf | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify) |
+| [MC-235035](https://bugs.mojang.com/browse/MC-235035) | Wenn man in einer benutzerdefinierten Dimension schläft, während "natural" ausgestellt ist, stürzt das Spiel ab    | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify) |
+| [MC-237493](https://bugs.mojang.com/browse/MC-237493) | Telemetrie kann nicht ausgestellt werden                                               | [TieFix](https://www.curseforge.com/minecraft/mc-mods/tiefix) |
+| [MC-249059](https://bugs.mojang.com/browse/MC-249059) | Ladefenster für die Landschaft schließt sich nicht bevor 2 Sekunden vergangen sind           | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify) |
+
+Falls du mehr Fehlerbehebungen für das Spiel aktivieren möchtest, schau dir gerne die folgenden Listen für [Debugify](https://github.com/W-OVERFLOW/Debugify/blob/1.18/PATCHED.md#unpatched-in-vanilla) und [TieFix](https://github.com/j-tai/TieFix#bugs-fixed) an.
+
+#### Eine Fehlerbehebung anfordern
+
+Willst du, dass ein Fehler in Fabulously Optimized behoben wird? Folge diesen Schritten:
+
+1. Finde den eigentlichen Fehler [in Mojangs offizieller Fehlersuche](https://bugs.mojang.com/projects/MC/issues?filter=allopenissues)
+2. Frage ihn an oder überprüfe seinen Status auf [Debugifys Fehlerübersicht](https://github.com/W-OVERFLOW/Debugify/issues)
+3. Sobald der Fehler in Debugify hinzugefügt wurde (oder dies geplant ist), kannst du [eine Anfrage zur Ändern der Einstellungen von FO](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/new?assignees=\&labels=option\&template=setting-request.yml) senden
+4. Beschreibe das Problem, sage, welche Mod dieses Problem beheben könnte und warum das Problem für zirka 70-80% der Nutzer von *FO* relevant ist.
+5. Wenn dein Fehler (*Issue*) akzeptiert wurde, dann kannst du dich darauf freuen, dass wahrscheinlich schon in der nächsten Version von *FO* behoben ist!
+
+### Mods konfigurieren
+
+Das Modpack ist bereits so vorkonfiguriert, um das bestmögliche Spielerlebnis zu bieten. 
+
+Falls du allerdings etwas ändern möchtest, kannst du das z.B. bei `Options...` → `Video Settings...` tun. Falls du einen sehr schnellen Computer und Monitor hast, kannst du dort `vSync` deaktivieren.
+
+Für andere Dinge wie dynamische Lichter, Grass-Verbesserungen und Zoom:
+
+1. Klicke auf `Mods`
+2. Ließ die Beschreibung, um zu wissen, was welche Mods tun
+3. Falls ein blauer Stift zu sehen ist, dann kannst du die Mod konfigurieren, indem du den dafür vorgesehenen Knopf drückst: ![](https://i.ibb.co/j35cBtn/image.png)
+ * Falls bei keinen Mods ein blauer Stift sichtbar ist, hast du wahrscheinlich aus Versehen das *Mod Menu Helper* Ressourcenpaket deaktiviert oder gelöscht. In diesem Fall:
```

Comment:
Indentation seems to have gone wrong here (check the spaces)

Reply:
Fixed! Thanks.

Reply:
The asterisk still needs one or more additional spaces

Reply:
lol you changed the English one

Reply:
ah whatever I'll merge and fix myself